### ISSUE TITLE
Cannot install MB-System using home-brew

### ISSUE THREAD

**Issue by rastamauli** [dwcaress/MB-System#1355]
Hi All

I got some problem during MB-System installation processed using "Homebrew".
The process stopped during downloaded OTPS.
It said :
" Error: mbsystem: Failed to download resource "otps"
Download failed: ftp://ftp.mbari.org/pub/Caress/OTPS_202107.tar.gz"
I'm using MacBook M1 Pro and OS Ventura 13.2.1

Thank You

**Comment by sbfreddie** [comment]
Yes, I have been getting the same error using Home-brew for quite some time.
Is there a fix for this??

Regards,
Ed

**Comment by jonathan-k4** [comment]
It looks like all the directories in the ftp site are empty.
